project/Easy-Travel-BD/EasyTravel.Web/wwwroot/assets/js/main.js

```
@@ -0,0 +1,218 @@
+   */
+  const preloader = document.querySelector('#preloader');
+  if (preloader) {
+    window.addEventListener('load', () => {
+      preloader.remove();
+    });
+  }
+
+  /**
+   * Scroll top button
+   */
+  let scrollTop = document.querySelector('.scroll-top');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'scrollTop' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)